app/src/pages/inside/testPlansPage/milestones/milestoneModals/createMilestoneModal/createMilestoneModal.scss

```
@@ -1,5 +1,27 @@
 @use '../../milestonesTable/milestonesShared' as ms;
 
+.milestone-type-dropdown {
+  :global(div[role='button'][class*='_dropdown_']) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)